YesterdayApi/Web/Cors/CorsExtension.cs

```diff
@@ -0,0 +1,18 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace YesterdayApi.Web.Cors
+{
+  public static class CorsExtension
+  {
+    public static void AddCorsPolicy(this IServiceCollection services)
+    {
+      services.AddCors();
+    }
+
+    public static void UseCorsPolicy(this IApplicationBuilder app)
+    {
+      app.UseCors(builder => builder.WithOrigins("http://yesterday.com"));
```

Comment:
As an open source project we should be open for every request from any client. Not from specific one as you did.

Hint:
            .AllowAnyOrigin()
            .AllowAnyMethod()
            .AllowAnyHeader()
            .AllowCredentials());